lms/resources/default.py

```diff
@@ -7,9 +7,7 @@ class DefaultResource:
 
     __acl__ = [
         (Allow, "report_viewers", "view"),
-        (Allow, "lti_user", "canvas_api"),
-        (Allow, "lti_user", "lti_outcomes"),
-        (Allow, "lti_user", "sync_api"),
+        (Allow, "lti_user", "api"),
```

Comment:
Yep. Seems better. If you get them all why distinguish? We can always go back and add distinctions if we ever get different user classes.